src/applications/ivc-champva/shared/utilities.js

```diff
@@ -138,3 +138,18 @@ export function adjustYearString(data) {
   });
   return copy;
 }
+
+/**
+ * Combine all street fields from an address into a single string.
+ * @param {Object} addr Standard form address object containing one or more `street` properties (e.g., street, street1, street2)
+ * @returns String of all street fields combined.
+ */
+export function concatStreets(addr) {
+  let res = '';
+  if (addr) {
+    for (const [k, v] of Object.entries(addr)) {
+      res += k.includes('street') ? `${v} ` : '';
+    }
+  }
+  return res;
+}
```

Comment:
Main contribution is this small helper to combine street key values into a single string